src/containers/AccountSettings/accountSettings.jsx

```diff
@@ -11,6 +10,19 @@ import NavigationBar from '../../components/navigationBar';
 import './accountSettings.css';
 
 const accountSettings = ({ info, handleLogOut, loading }) => {
+  const redirect = () => {
+    if (!info.createdByGoogle && !info.createdByTwitter) {
+      return '/confirm-password';
```

Comment:
do not use the static route like this | use the constants route
```suggestion
      return CONFIRM_PASSWORD;
```